docs/4.0/components/breadcrumb.md

```diff
@@ -32,3 +32,22 @@ Similar to our navigation components, breadcrumbs work fine with or without the
   <span class="breadcrumb-item active">Bootstrap</span>
 </nav>
 {% endexample %}
+
+
+## Regarding accessibility
```

Comment:
This would be the only heading on the page; can we change it to `## Accessibility` for brevity? And then also add an `## Overview` or `## Example` above line 8?